### ISSUE TITLE
Mocha failing after update to 3.10.0

### ISSUE THREAD

**Issue by davidfilat** [dividab/tsconfig-paths#166]
![image](https://user-images.githubusercontent.com/1475163/124637041-e9db0000-de91-11eb-913f-96924ab1d8cd.jpeg)

**Comment by davidfilat** [comment]
Here is my .mocharc
```

allow-uncaught: false
async-only: true
color: true
diff: true
exit: true
extension:
  - 'ts'
file:
  - './src/controller/__tests__/init-db-connection.ts'
forbid-only: false
forbid-pending: false
full-trace: false
growl: false
recursive: false
reporter: 'spec'
retries: 0
slow: 100
sort: false
spec: './src/**/*.spec.ts'
timeout: 5000
trace-warnings: true
v8-stack-trace-limit: 100 # V8 flags are prepended with "v8-"
watch: false
require:
  - 'dotenv/config'
  - 'ts-node/register'
  - 'tsconfig-paths/register'


```

**Comment by jonaskello** [comment]
@davidfilat I just published a fix for this in 3.10.1, could you confirm if it works?

**Comment by davidfilat** [comment]
It works! Thank you!
